app.css

```diff
@@ -172,7 +172,8 @@ body {
       font-size: .9em;
       white-space: nowrap;
       overflow: hidden;
-      text-overflow: ellipsis; }
+      text-overflow: ellipsis;
+      width: 240px; }
```

Comment:
I think there is a good chance it's gonna mess up with the flex column of Bulma, did you test the responsive render on mobile?

Reply:
I will test it as soon as I'm at home.

Reply:
Sry for the late response. I think it's working:

![Screenshot1](https://user-images.githubusercontent.com/7312638/77080242-5d38d400-69f9-11ea-86f4-41fddfd4bc91.jpg)

![Screenshot2](https://user-images.githubusercontent.com/7312638/77080259-61fd8800-69f9-11ea-8e37-44a7af04e892.jpg)